server/ocsp.go

```diff
@@ -558,10 +558,11 @@ func (s *Server) configureOCSP() []*tlsConfigKind {
 			tlsConfig: config,
 			tlsOpts:   opts,
 			apply: func(tc *tls.Config) {
-
 				// RequireAndVerifyClientCert is used to tell a client that it
 				// should send the client cert to the server.
-				tc.ClientAuth = tls.RequireAndVerifyClientCert
+				if opts.Verify {
```

Comment:
Will `opts.Verify` reflect `verify:` and `verify_and_map:` TLS config block entries?

Reply:
Yes verify will be enabled if `verify_and_map` is enabled.